### DISCUSSION TITLE
Automated Valve Reccomendation?

### DISCUSSION THREAD

**Discussion by johnny2678** [tagyoureit/nodejs-poolController#411] [Q&A] [answered]
Discussions!  Hope I'm using this right ;)

I think my Pool/Spa drain actuator is shot.  Walked out last night to find all the spa water in the pool and the spa drain still on, even though pool was in pool mode.  I tried manually flipping the switch between positions but no motor activity at all.  It's one of those [CVA-24 Pentair models](https://www.amazon.com/Pentair-Compool-Replacement-ACTUATOR-ROTATION/dp/B002CTLTFC) 

They are a pain because there is no way to manually turn these without removing the motor from the valve, which is what I did to flip the drain back to Pool.

Reading through the[ Nixie guide](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#automated-valves) , I didn't see a recommendation for Valves - would the[ intermatic ones](https://www.amazon.com/Intermatic-PE24VA-Valve-Actuator-Black/dp/B002ZPJVV2/ref=sr_1_1?crid=VNHN5TDL46MP&keywords=intermatic+valve&qid=1649679264&s=lawn-garden&sprefix=intermatic+valve%2Clawngarden%2C85&sr=1-1) work ok?  Much cheaper...

**Comment by rstrouse** [comment, marked as answer]
I have 8 automated valves on my pool.  I started out with CVA-24 valves and ended up replacing or repairing each one of them twice over the years.  A few years ago I switched them all to IntelliValves and have never had a disruption since...  Knock on PVC.  When is service mode you can move these to whatever position you want.

All that said, most 24vac valves are interchangeable and you can use them as direct replacements.   Unfortunately, with the exception of IntelliValve they all contain the same components that fail.  The two micro switches and cam assemblies are the failure points in almost every case this was the problem.  Only once did I have a motor failure.

**Comment by johnny2678** [comment]
Ding Ding Ding - just did some playing at the pad.  When I open up the faulty valve and manually move the plastic tension clip, it actuates back to pool mode.  Flip the switch to position 1 and it actuates back to spa mode, but doesn't actuate back to pool mode without manually moving the clip.  

I ordered the intermatic one above as a replacement - doesn't hurt to have one on standby.  

Will do some reading on intellivalves - but I don't have an intellicenter?

EDIT: Ooo - automated partial valve opening/closing - could really use this on my spillway!!  and another on my skimmer/main drain return valve.  I could automate closing the skimmer off when doing maintenance so it doesn't drain back to the pool and cause the pump to lose prime.  Does njspc support these standalone (no OCP or easytouch)?

**Comment by rstrouse** [comment]
IntelliValve operates just like any other 24vac valve actuator and does not require IntelliCenter.  What it does is remove the microswitches and mechanical cams that are the point of failure and replaces them with a more precise valve position sensor.  Since I have replaced all my actuators I have not had a failure and it always returns to the same position on each endpoint.  It also allows you to move the valve without removing it by pressing the left or right buttons.

If Pentair ever gets around to getting the RS485 hooked up to these we will see if we can integrate them into Nixie.

**Comment by rock-crusher** [comment]
Here’s a little tip that may help some:
Many people are unaware that on some auto valve actuators you can manually rotate the valve position without having to remove it. Simply back out the handle lock knob a few turns then push in on the knob itself and it will disengage the gears allowing you to rotate the handle manually putting the valve in any position you like.
The tiny 3 position electric switch on the end of the actuator body should be placed in the off position when doing this.
Not sure if this works for all motorized actuators, but worth a try.
 Hope this helps some of you 😎

**Reply by johnny2678** [reply]
Thanks.  I can try that trick but II think mine (CVA-24 - pentiar OEM) can't be rotated manually like some others can.  I did order some of those microswitches to see if I can get it working again.  Will swap this one out for the new one I ordered but if I can do do surgery on it then maybe i'll keep as a backup.

Until Pentiar adds rs485 to Intellivalve ;)

**Reply by johnny2678** [reply]
btw, intellivalve just dropped $30 on Amazon - https://www.amazon.com/Pentair-521485-IntelliValve-Valve-Actuator/dp/B01BK7L2A8/ref=sr_1_2?crid=5CVQYMHEURTU&keywords=intellivalve&qid=1649764514&sprefix=intellivalve%2Caps%2C107&sr=8-2
